src/manual-edits/content/docs/new-relic-only/basic-style-guide/writing-guidelines/docs-translation.mdx

```diff
@@ -0,0 +1,195 @@
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Add another item
+          </td>
+
+          <td>
+            Add another block for a target language. Currently not used.
+          </td>
+        </tr>
+      </tbody>
+    </Table>
+  </Collapser>
+
+  <Collapser
+    id="view-any"
+    title="View the translation status for any doc"
+  >
+    To view the translation status for **any** doc:
+
+    1. From the Drupal Docs site, go to **My workbench > Translation index**.
+    2. Use any of the available search options, then select **Apply**.
+    3. Optional: Select any doc to view or edit it directly.
+
+    ![Drupal Translation index](./images/drupal-translation-index.png "Drupal Translation index")
+
+    To search which docs have been translated or have wish list status: From your Drupal administration page, select **My workbench > Translation index**.
+  </Collapser>
+</CollapserGroup>
+
+## Request translation
+
+New Relic uses a translation services company to translate docs. The original contract in 2018 approved 200 key docs specifically selected for the Japanese market only. Additional requests to translate docs require budget approval from the translation project team and the New Relic sales manager for Japan. Requests for other markets and languages will be addressed in the future.
+
+To request a doc for translation:
+
+1. Make sure the doc meets criteria for translation, such as:
+   * High traffic, particularly for that audience
+   * Strategic importance to company
+   * Introductory and conceptual docs that tell a valuable story about the product or feature
+   * Demonstrated customer interest in that market (for example, the Windows/.NET stack is particularly important in Japan)
+   * Allows us to tell a "complete" story (for example, including all installation docs for a particular language so we can offer a complete experience)
+2. [File a DOC JIRA](http://docs.newrelic.com/jira) outlining your business case.
+
+The Tech Docs team will follow up with you for additional details, forward your request to the translation project team, and notify you when your doc has been translated.
+
+## Update translated doc [#update-translation]
+
+Every two weeks the translation services company automatically runs a program that compares changes between the English and Japanese sites. If a translated doc has changes, typically it will be re-translated. If budget considerations affect updates to a translated doc, the translation project team will work with the Tech Docs team and the requester to approve and prioritize the update.
+
+## Analyze traffic to translated docs [#translation-insights]
+
+To track traffic on the Japanese Docs site, use any of these options:
+
+* [Google Analytics](https://analytics.google.com/analytics/web/?hl=en&pli=1#/realtime/rt-overview/a3047412w51432103p183513476/)
+* [Insights dashboard](https://insights.newrelic.com/accounts/727059/dashboards/807464)
```

Comment:
Maybe we should replace the Insights dashboards since its EoL is coming.

Reply:
Good call.